src/electionguard-ui/ElectionGuard.UI/Models/TallyCeremonyChecklist.cs

```diff
@@ -5,21 +5,26 @@ public partial class TallyCeremonyChecklist : ObservableObject
     [ObservableProperty]
     private bool _quorumReached;
 
-    public bool TallyStarted => State >= TallyState.TallyStarted;
-    public bool SubtaliesCombined => State >= TallyState.PendingGuardianDecryptShares;
-    public bool AllDecryptionSharesComputed => SubtaliesCombined && (
-        State > TallyState.PendingGuardianDecryptShares ||
-        (State == TallyState.PendingGuardianDecryptShares && _sharesComputed >= _quorum));
+    [ObservableProperty]
```

Comment:
Converted all of these fields to observable properties to be seen better by the bindings